source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,79 +1,77 @@
-    <div class='ca_center ca_credit'>CHBMB, SpaceInvaderOne, Sparklyballs, wgstarks, DJoss, Zer0Nin3r, Mex, prostuff1, bonienl, ljm42, kizer, trurl, Jos, Lime Technology Inc, SimonF, ich777, jimmy898, Alex.b, neruve, Eugeni_CAT, ChaseCares, TheEyeTGuy, dlandon</div>
-    <div class='ca_center ca_creditheader'><?tr("Additional Libraries");?></div>
-    <div class='ca_center ca_credit'>Awesomeplete (Lea Verou)<br>Chart.js (various)<br>chartjs-plugin-trendline (Marcus Alsterfjord)<br>XML2Array/Array2XML (Miles Johnson)<br>Magnific-Popup (Dmytro Semenov)</div>
-    <br>
-    <div class='ca_center ca_italic'><font size='1rem;'><?tr("Copyright");?> &copy; 2020-2025 Lime Technology, 2015-2025 Andrew Zawadzki</font></div>
+      <ul class='caMenu'>
+      <span id='category_list' class='onlyShowWithFeed'></span>
+      <li class='categoryMenu caMenuItem allApps onlyShowWithFeed nonDockerSearch' data-category='All'><?tr("All Apps");?></li>
+      <hr class='category_hr'>
+      <li class='categoryMenu caMenuItem caRepositoryMenu onlyShowWithFeed nonDockerSearch' data-category='repos'><?tr("Repositories");?></li>
+    <hr class='category_hr onlyShowWithFeed'>
+      <li class='caMenuItem' onclick='window.location="/Apps/ca_settings"'><?tr("Settings");?></a></li>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)